src/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -132,6 +136,128 @@ public void Uri_Relative_BaseVsSimplePartialPathWithScheme_ReturnsPartialPathWit
         }
 
         [Fact]
+        public void Uri_Relative_SimplePartialPathWithScheme_Unicode_ReturnsPartialPathWithScheme()
+        {
+            string schemeAndRelative = "http:Ç";
```

Comment:
Nit: Sorry for not catching this earlier: please replace the Unicode character with the numerical equivalents \xnnnn. This is something we tried to do with the rest of the code.